src/requests/api.py

```diff
@@ -72,6 +72,8 @@ def get(url, params=None, **kwargs):
 
     return request("get", url, params=params, **kwargs)
 
+def send(url, **kwargs):
+    return request("send", url, **kwargs)
```

Comment:
**Suggestion:** The new 'send' method in api.py is not following the HTTP methods convention. Consider renaming it to a proper HTTP method or make it more generic if it's intended to handle multiple methods.
```suggestion
def custom_request(method, url, **kwargs):
    return request(method, url, **kwargs)
```